src/js/components/SelectMultiple/index.d.ts

```diff
@@ -8,6 +8,23 @@ export interface SelectMultipleProps extends BasicSelectProps {
   value?: (string | number | object)[];
   showSelectedInline?: boolean;
   sortSelectedOnClose?: boolean;
+  messages?: {
+    clearAll?: string;
+    clearAllA11y?: string;
+    multiple?: string; // kept for backwards compatibility, but summarizedValue should be used instead
+    onMore?: string;
```

Comment:
```suggestion
    showMore?: string;
```
this should be showMore?